docs/source/docs/quick-start/quick-install.md

```diff
@@ -0,0 +1,38 @@
+# Quick Install
+
+## Install the latest image of photonvision for your coprocessor
+
+- For the supported coprocessors
+  - RPI 3,4,5
+  - Orange Pi 5
+  - Limelight
+
+For installing on non-supported devices {ref}`see. <docs/advanced-installation/sw_install/index:Software Installation>`
+
+[Download the latest preconfigured image of photonvision for your coprocessor](https://github.com/PhotonVision/photonvision/releases/latest)
+
+| Coprocessor          | Image filename                                       | Jar                                   |
+| -------------------- | ---------------------------------------------------- | ------------------------------------- |
+| OrangePi 5           | photonvision-{version}-linuxarm64_orangepi5.img.xz   | photonvision-{version}-linuxarm64.jar |
+| Raspberry Pi 3, 4, 5 | photonvision-{version}-linuxarm64_RaspberryPi.img.xz | photonvision-{version}-linuxarm64.jar |
+| Limelight 2          | photonvision-{version}-linuxarm64_limelight2.img.xz  | photonvision-{version}-linuxarm64.jar |
+| Limelight 3          | photonvision-{version}-linuxarm64_limelight3.img.xz  | photonvision-{version}-linuxarm64.jar |
+
+:::{warning}
+Balena Etcher 1.18.11 is a known working version. Other versions may cause issues such as bootlooping (the system will repeatedly boot and restart) when imaging your device.
+:::
+
+Use the 1.18.11 version of [Balena Etcher](https://github.com/balena-io/etcher/releases/tag/v1.18.11) to flash the image onto the coprocessors micro sd card. Select the downloaded `.img.xz` file, select your microSD card, and flash.
+
+Limelights have a different installation processes. Simply connect the limelight to your computer using a usb c cable. Select the compute module. If it doesn’t show up after 30s try using another USB port, initialization may take a while. If prompted, install the recommended missing drivers. Select the image, and flash.
```

Comment:
limelights have different versions, is this true for all?

Reply:
For the ll3g it is different however for all the "supported" models it is true.